bsp-gonk/board/victara/patch/build/0004-Add-mechanism-to-disable-dependencies-for-the-otapac.patch

```diff
@@ -0,0 +1,45 @@
+From 6b74dba481f079fd5e3713979b45c3f1df7db5b3 Mon Sep 17 00:00:00 2001
```

Comment:
I vote to remove this patch.  It's really just a hack to avoid running out of disk space on Travis when building an OTA update package for the board.  We do this for our own purposes on Kenzo but Victara likely won't have this problem